src/components/inputs/InputText/index.tsx

```diff
@@ -1,11 +1,49 @@
 // Dependencies
-import React from "react";
+import React, { useCallback, useState } from "react";
 
 // Styled Components
-import * as styled from "./styles";
+import { Layout, InputContainer, InputValue, Side, Label } from "./styles";
+import { InputTextProps } from "./types";
 
-function InputText() {
-  return <styled.Layout>WIP</styled.Layout>;
+import ErrorMessage from "components/display/ErrorMessage";
+
+function InputText(
+  {
+    error = null,
+    disabled = false,
+    placeholder = "Please enter a value",
+    label = "Withdraw address",
```

Comment:
We need to add onChange, onBlur, name, ref to work with react-hook-form. See https://github.com/zignaly-open/zignaly-ui/pull/35

Reply:
Ok, will do